### PR TITLE
FIX: The editor hover window doesn't show in the test cases #506

### PR DESCRIPTION
**Pull request by oussamaBGZ**
i fixed the problem with the hover its just one line of code :) #506

### CODE REVIEW

**Comment by bautistaaa** [comment]
we have a PR open to move it to the bottom current so this isnt needed!

**Comment by bautistaaa** [comment]
actually this is probably helpful. will add in for now.

**Review by bautistaaa — APPROVED** [review]

**Comment by bautistaaa** [comment]
can you run `pnpm checks:fix` youre failing prettier

**Comment by oussamaBGZ** [comment]
@bautistaaa i had to fix some linting issues the ways "typeof import('monaco-editor')" was implemented caused a warning

File: packages/monaco/src/code-editor.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { useTheme } from 'next-themes';
 import { useMemo } from 'react';
 import { useEditorSettingsStore } from './settings-store';
 import { libSource } from './editor-types';
+import type * as MonacoEditor from 'monaco-editor';
```
Comment (bautistaaa) [inline packages/monaco/src/code-editor.tsx:8]: do you need the type keyword in addition to typeof of below?

Comment (oussamaBGZ) [reply packages/monaco/src/code-editor.tsx:8]: Technically we don't but Eslint will throw a warning:
"All imports in the declaration are only used as types. Use `import type`."

Comment (bautistaaa) [reply packages/monaco/src/code-editor.tsx:8]: ah ok it’s ok to eslint ignore when needed

**Review by bautistaaa — APPROVED** [review]

File: packages/monaco/src/code-editor.tsx (hunk 1 above)
Comment (oussamaBGZ) [reply packages/monaco/src/code-editor.tsx]: ok done

**Pull request merged**